christin/index.js

```diff
@@ -0,0 +1,77 @@
+//Programming Basics: Nested Loops(Using Arrays)
+//Use NESTED LOOPS in each instance
+
+/*Write a program to construct the following pattern:
+ *
+ * *
+ * * *
+ * * * *
+ */
+
+let star = ["*"];
+let starSum = "";
+
+function stars(array) {
+  for (let item of array) {
+    for (let i = 0; i < 5; i++) {
+      starSum += item;
+    }
+  }
+  return starSum;
+}
+
+console.log(stars(star));
+
+//2. Write a program which prints the elements of the following array:
+// let arr = [[1, 2, 1, 24], [8, 11, 9, 4], [7, 0, 7, 27], [7, 4, 28, 14], [3, 10, 26, 7]];
+//And creates the following output:
+
+// row 0
+// 1
+// 2
+// 1
+// 24
+// row 1
+
+let arrAllNumbers = [
+  [1, 2, 1, 24],
+  [8, 11, 9, 4],
+  [7, 0, 7, 27],
+  [7, 4, 28, 14],
+  [3, 10, 26, 7],
+];
+
+let arrNumber = [];
+
```

Comment:
Task 2 ==> You get no output in terminal because the function name of task 2 and 3 are same!
So now if you change the name you will get output but the result which prints is =>
 row 0
 [ 1, 2, 1, 24 ]
You should also print other rows 
row 1
8, 11, 9, 4  and so on...
Here some tips to fix the code =>
- In line 44 change the empty array to an empty string
- Get rid of the code that you have within outer for loop and instead of it you need   for (i = 0 ; i < arr.length; i++) 
- Get rid of console.log() which is inside of outer loop
- We declare a new variable inside of outer loop name it for example let indexArr = arr[i]
- Then in next line we add i to our empty string that we declared before function means arrNumber += `Row ${i} \n`
- Now we have to modify the inner loop, therefore get rid of what you have within for loop and write 
    for (number of indexArr) 
- Get rid of arrNumber.push(number) and try to add arrNumber += `${number} \n` instead, so now we also have access to numbers
- The last step : move the return before the last curly braces }

Reply:
i added that